src/main/java/calculator/parser/SentenceParser.java

```diff
@@ -0,0 +1,42 @@
+package calculator.parser;
+
+import calculator.Operand;
+import calculator.Operator;
+
+public abstract class SentenceParser<T> {
+    public Operand<T> parse(String sentence){
+        String[] splitted = sentence.split(" ");
+        Operand<T> firstOperand = operandFromValue(fromString(splitted[0]));
+        Operand<T> lastOperand = firstOperand;
+        Operator<T> lastOperator = operatorFromSign(Operator.Sign.fromSignStr(splitted[1]));
+        boolean isOperand = true;
+        connect(lastOperand, lastOperator);
+        for(int i=2;i<splitted.length;i++){
+            String s = splitted[i];
+            if(isOperand){
+                lastOperand = operandFromValue(fromString(s));
+                connect(lastOperator, lastOperand);
+                isOperand = !isOperand;
```

Comment:
```suggestion
                isOperand = false;
```
if 조건문에서 true이기 때문에 들어온 isOperand이기때문에 !isOperand라고 값을 변경하는 것보다는 false로 정의하는게 좋을 듯 합니다.

Reply:
반전의 의미를 주고자 이렇게 작성했는데 false가 더 명확한 것 같네요!